src/CoreOperators.ts

```diff
@@ -15,7 +15,7 @@ export interface CoreOperators<T> {
   combineAll?: <R>(project?: (...values: Array<any>) => R) => Observable<R>;
   combineLatest?: <R>(...observables: Array<Observable<any> | ((...values: Array<any>) => R)>) => Observable<R>;
   concat?: <R>(...observables: (Observable<any> | Scheduler)[]) => Observable<R>;
-  concatAll?: () => Observable<any>;
+  concatAll?: () => Observable<T>;
```

Comment:
I was actually thinking about `concat` as well, return type may need to be `Observable<any>` instead of specific types. I think there would be possible cases of `concatall` executed over multiple observable has different types - what do you think?